sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -356,6 +356,33 @@ struct MemAllocRecord : _pi_object {
   bool OwnZeMemHandle;
 };
 
+// Struct used to fetch device wall-clock time
+struct piDeviceTime {
```

Comment:
why do you need this class (constructed for device) instead of having its contents into pi_device itself?